docs/modules/druid/examples/getting_started/druid.yaml

```diff
@@ -23,13 +23,13 @@ spec:
       default:
         replicas: 1
     roleConfig:
-      listenerClass: external-stable # This exposes this role outside of Kubernetes. Remove this configuration if this is not desired
```

Comment:
I honestly think the previous version was more correct.
The listenerClass is not a per-stacklet (as it was earlier) but a per-role config